kousa/env.txt

```diff
@@ -0,0 +1,16 @@
+export DATABASE_URL=postgres://postgres:password@localhost/kousa_repo2
+export BEN_GITHUB_ID=7872329
+export RABBITMQ_URL=amqp://user:password@localhost:5672
+export ACCESS_TOKEN_SECRET=
+export REFRESH_TOKEN_SECRET=
+export GITHUB_CLIENT_ID=
+export TWITTER_API_KEY=
+export TWITTER_SECRET_KEY=
+export TWITTER_BEARER_TOKEN=
+export GITHUB_CLIENT_SECRET=
+export SENTRY_DNS=
+export API_URL=http://localhost:4001
+export WEB_URL=http://localhost:3000
+export PORT=4001
+export DISCORD_CLIENT_ID=829620188831678474
+export DISCORD_CLIENT_SECRET=wOM9loTVDNOStJsaIH51mr64BIBJCBSp
```

Comment:
lmao

Reply:
reset ur discord

Reply:
oh no, well nvm it dont matter cuz its just for testang